tests/units/units.cpp

```diff
@@ -1249,8 +1351,9 @@ TEST(Units, checkScalingFactorOneUnitImported)
 
     u1->setImportSource(import);
 
-    EXPECT_EQ(1.0, libcellml::Units::scalingFactor(u1, u2));
-    EXPECT_EQ(1.0, libcellml::Units::scalingFactor(u2, u1));
+    // KRM Changed to be 0.0 because the units u1 contain an import?
+    EXPECT_EQ(0.0, libcellml::Units::scalingFactor(u1, u2));
+    EXPECT_EQ(0.0, libcellml::Units::scalingFactor(u2, u1));
```

Comment:
Is that second test really needed (since we are returning `0.0`)?

Reply:
For completeness, the reverse arrangement is tested too.